### PR TITLE
EF6 with ASP.NET Core -- project.json to .csproj

### PR DESCRIPTION
**Pull request by tdykstra**
@spboyer @Rick-Anderson please review

### CODE REVIEW

File: aspnetcore/data/entity-framework-6.md [hunk 1]
```diff
@@ -16,126 +16,47 @@ uid: data/entity-framework-6
...
+The recommended way to use Entity Framework 6 in an ASP.NET Core application is to put the EF6 context and model classes in a class library project that targets the full framework. Add a reference to the class library from the ASP.NET Core project. See the sample [Visual Studio solution with EF6 and ASP.NET Core projects](https://github.com/aspnet/Docs/tree/master/aspnetcore/data/entity-framework-6/sample/).
 
-It's not feasible to put an EF6 context in an ASP.NET Core 1.0 project because *.xproj*-based projects don't support all of the functionality that EF6 commands such as *Enable-Migrations* require. In a future release of ASP.NET Core, Core projects will be based on *.csproj* files, and at that time you'll be able to include an EF6 context directly in an ASP.NET Core project.
+It's not feasible to put an EF6 context in an ASP.NET Core project because .NET Core projects in Visual Studio 2017 don't support all of the functionality that EF6 commands such as *Enable-Migrations* require.
```
Comment (Rick-Anderson) [inline aspnetcore/data/entity-framework-6.md]: I'm not a fan of feasible. Above OK but consider:
EF6 cannot be use in a  ASP.NET Core project.  .NET Core projects in Visual Studio 2017 don't support all of the functionality that EF6 commands such as *Enable-Migrations* require.

Also, why is it dependent on VS17?  You can use the CLI to run EF commands. Is it true to just say:

EF6 cannot be use in a  ASP.NET Core project.  .NET Core projects ~~in Visual Studio 2017~~ don't support all of the functionality that EF6 commands such as *Enable-Migrations* require.

**Comment by Rick-Anderson** [comment]
@tdykstra  please supply review link. https://review.docs.microsoft.com/en-us/aspnet/core/data/entity-framework-6?branch=pr-en-us-2826 doesn't seem to be the right branch.

File: aspnetcore/data/entity-framework-6.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-﻿---
+---
 title: Getting Started with ASP.NET Core and Entity Framework 6 | Microsoft Docs
 author: rick-anderson
```
Comment (Rick-Anderson) [inline aspnetcore/data/entity-framework-6.md]: change to u

File: aspnetcore/data/entity-framework-6.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-description: 
-keywords: ASP.NET Core,
+description: This article shows how to use Entity Framework 6 in an ASP.NET Core application.
+keywords: ASP.NET Core, Entity Framework, EF 6
 ms.author: riande
```
Comment (Rick-Anderson) [inline aspnetcore/data/entity-framework-6.md]: ditto

File: aspnetcore/data/entity-framework-6.md [hunk 4]
```diff
@@ -16,126 +16,47 @@ uid: data/entity-framework-6
...
-    "net452": {
-      "dependencies": {
-        "EF6": {
-          "target": "project"
-        }
-      }
-    }
-  },
-```
+Your ASP.NET Core project has to reference the full .NET framework and EF6. For example, the *.csproj* file of your ASP.NET Core project will look similar to the following example (only relevant parts of the file are shown).
```
Comment (Rick-Anderson) [inline aspnetcore/data/entity-framework-6.md]: maybe
The ASP.NET Core project *needs* to ref the full ....

**Comment by tdykstra** [comment]
Review link https://review.docs.microsoft.com/en-us/aspnet/core/data/entity-framework-6?branch=ef6 

After merge: https://review.docs.microsoft.com/en-us/aspnet/core/data/entity-framework-6?branch=csproj

**Pull request merged**